db/migrate/20220409131727_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string 'name'
+      t.string 'photo'
+      t.string 'bio'
```

Comment:
- [ ] (Optional) It would be nice to make the `bio` a type of `text` instead of a `string.`